.github/workflows/c++-code-formatting.yml

```diff
@@ -160,28 +160,39 @@ jobs:
 
       - name: Check copyright headers
         env:
-          # The expected copyright notice.
+          # The expected copyright notice. Comment markers ("//" or "#") are
+          # added automatically and should NOT appear in this variable.
           COPYRIGHT: |-
-            // Copyright 2019-2020 CERN and copyright holders of ALICE O2.
-            // See https://alice-o2.web.cern.ch/copyright for details of the copyright holders.
-            // All rights not expressly granted are reserved.
-            //
-            // This software is distributed under the terms of the GNU General Public
-            // License v3 (GPL Version 3), copied verbatim in the file "COPYING".
-            //
-            // In applying this license CERN does not waive the privileges and immunities
-            // granted to it by virtue of its status as an Intergovernmental Organization
-            // or submit itself to any jurisdiction.
+            Copyright 2019-2020 CERN and copyright holders of ALICE O2.
+            See https://alice-o2.web.cern.ch/copyright for details of the copyright holders.
+            All rights not expressly granted are reserved.
+
+            This software is distributed under the terms of the GNU General Public
+            License v3 (GPL Version 3), copied verbatim in the file "COPYING".
+
+            In applying this license CERN does not waive the privileges and immunities
+            granted to it by virtue of its status as an Intergovernmental Organization
+            or submit itself to any jurisdiction.
         run: |
-          # Find changed C++ files.
+          # Find changed C++ and CMake files. Keep the file extensions in sync
+          # with the ones in the "case" statement below!
           readarray -d '' files < \
-            <(git diff -z --diff-filter d --name-only --merge-base \
-                  "$BASE_SHA" -- '*.cxx' '*.h')
-          # Run copyright notice check
+            <(git diff -z --diff-filter d --name-only --merge-base "$BASE_SHA" \
+                       -- '*.cxx' '*.h' '*.C' '*.cmake' CMakeLists.txt)
+          # Run copyright notice check. Comment lines start with "//" for C++
+          # files and "#" for CMake files.
+          cpp_copyright=$(echo "$COPYRIGHT" | sed -r 's,^.+,// \0,; s,^$,//,')
```

Comment:
While you are at it. Can you also ignore the dates for the copyright?

Reply:
Done in 15dd00de8a73d3f599022b12e0a0e0ddd8d079ce!